deploy/olm-catalog/argocd-operator/0.2.0/argoproj.io_argocds.yaml

```diff
@@ -4336,6 +4336,8 @@ spec:
                   SSO providers are configure in CR. Unknown: For some reason the
                   SSO configuration could not be obtained.'
                 type: string
+              url:
```

Comment:
This file is obsolete, was it updated by the `make bundle` target ?

Reply:
yes it was done automatically by `make bundle`, am I on the wrong version or something? Maybe I'm missing some newer changes and need to rebase?

Reply:
this file is obsolete. No need to worry w.r.t this PR. I will take care of removing it in a different PR.